app.py

```diff
@@ -6,7 +6,17 @@
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    items = session.get_items()
+    session.add_item('Avi')
```

Comment:
This line means that every time you load the page, a new item "Avi" gets saved.

Reply:
ok got it.. will incorporate the change